.github/workflows/build.yml

```diff
@@ -58,7 +68,7 @@ jobs:
     runs-on: ${{ matrix.os}}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-13]
+        os: [ubuntu-latest, windows-latest, macos-12]
```

Comment:
test this on -12 and -13. But in practice I doubt it matters much.

Reply:
Added in https://github.com/aforren1/ptb-wheels/pull/14